pkg/bootstrapper/bootstrapper.go

```diff
@@ -132,6 +129,16 @@ type winNodeBootstrapper struct {
 	cni *cniOptions
 }
 
+// service struct contains the service information
+type service struct {
+	// svcObj is a pointer to the Windows service object
+	svcObj *mgr.Service
+	// svcMgr is used to interact with the Windows service API
+	svcMgr *mgr.Mgr
+	// svcDependents contains a list of services dependent on the current service
+	svcDependents []string
+}
```

Comment:
This struct is written like it is meant for a generic service, however it includes the service manager within it. If we had multiple services, we end up with multiple service managers. IMO this struct should either become specific to the kubelet service, or have the manager moved out of it.